### PR TITLE
HV-1440 Provide patch file for WF 11.x

### PR DESCRIPTION
**Pull request by gsmet**
* https://hibernate.atlassian.net/browse/HV-1440

And also run the integration tests for WF 11.x.

### CODE REVIEW

File: integration/pom.xml [hunk 1]
```diff
@@ -239,39 +335,39 @@
                 <groupId>org.wildfly.plugins</groupId>
                 <artifactId>wildfly-maven-plugin</artifactId>
                 <executions>
-                    <!-- Currently the WF Maven plug-in cannot apply offline commands, although patch itself wouldn't require a running
-                    server; see https://issues.jboss.org/projects/WFMP/issues/WFMP-11 -->
-                    <execution>
-                        <id>start-wildfly-for-patching</id>
```
Comment (gunnarmorling) [inline integration/pom.xml]: Ah, nice we don't need that start/stop cycle any longer.

**Comment by gunnarmorling** [comment]
Just waiting for the PR builder finish, then I'll merge it.

**Comment by gsmet** [comment]
Jenkins, retest this please

**Comment by gsmet** [comment]
Jenkins, retest this please

**Comment by gsmet** [comment]
@gunnarmorling it's green now, had to upgrade the Maven version we use for the builds.

**Comment by gunnarmorling** [comment]
Applied. Thanks, @gsmet!

**Pull request merged**